### PR TITLE
Add absolute and relative GSL solver parameters

### PR DESCRIPTION
**Pull request by clinssen**

### CODE REVIEW

File: tests/nest_tests/test_aeif_integrator.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            'Delta_T': 0.8,
+            'tau_w': 144.0,
+            'V_th': -57.0
+        }
+
+        neuron = nest.Create("aeif_cond_exp_neuron_nestml")
+
+        for param in params:
+            neuron.set({param: params[param]})
+
```
Comment (med-ayssar) [inline tests/nest_tests/test_aeif_integrator.py:80]: Maybe doing this instead of the loop:
```suggestion
        neuron.set(**param)
```

Comment (clinssen) [reply tests/nest_tests/test_aeif_integrator.py]: Thanks for the tip! Done.

File: pynestml/codegeneration/nest_code_generator.py [hunk 2]
```diff
@@ -109,6 +109,7 @@ class NESTCodeGenerator(CodeGenerator):
         - **module_templates**: A list of the jinja templates or a relative path to a directory containing the templates related to generating the NEST module.
     - **nest_version**: A string identifying the version of NEST Simulator to generate code for. The string corresponds to the NEST Simulator git repository tag or git branch name, for instance, ``"v2.20.2"`` or ``"master"``. The default is the empty string, which causes the NEST version to be automatically identified from the ``nest`` Python module.
     - **solver**: A string identifying the preferred ODE solver. ``"analytic"`` for propagator solver preferred; fallback to numeric solver in case ODEs are not analytically solvable. Use ``"numeric"`` to disable analytic solver.
+    - **gsl_adaptive_step_size_controller**: For the numeric (GSL) solver: how to interpret the absolute and relative tolerance values. Can be changed to trade off integration accuracy with numerical stability. The default value is ``"with_respect_to_solution"``. Can also be set to ``"with_respect_to_derivative"``. (Tolerance values can be specified at runtime as parameters of the model instance.)
```
Comment (pnbabu) [inline pynestml/codegeneration/nest_code_generator.py:112]: Can we add a link to the GSL docs so that it's easier to understand what each value means?

Comment (clinssen) [reply pynestml/codegeneration/nest_code_generator.py]: Good idea! Done.

**Review by pnbabu — APPROVED** [review]
Looks good to me!

**Pull request merged**